File: src/java/src/quickfix/fix41/TestRequest.java
package quickfix.fix41;
import quickfix.FieldNotFound;
import quickfix.Group;
import quickfix.field.*;

public class TestRequest extends Message
{
  public TestRequest()
  {
    getHeader().setField(new MsgType("1"));
  }
  public TestRequest(
    quickfix.field.TestReqID aTestReqID ) {

    getHeader().setField(new MsgType("1"));
    set(aTestReqID);
  }

  public void set(quickfix.field.TestReqID value)
  { setField(value); }
  public quickfix.field.TestReqID get(quickfix.field.TestReqID  value)
    throws FieldNotFound
  { getField(value); return value; }
  public quickfix.field.TestReqID getTestReqID() throws FieldNotFound
  { quickfix.field.TestReqID value = new quickfix.field.TestReqID();
    getField(value); return value; }
  public boolean isSet(quickfix.field.TestReqID field)
  { return isSetField(field); }
  public boolean isSetTestReqID()
  { return isSetField(112); }
}

File: src/.NET/fixt11/TestRequest.cs
namespace QuickFixT11
{

  public class TestRequest : Message
  {
    public TestRequest() : base(MsgType()) {}
    static QuickFix.MsgType MsgType() { return new QuickFix.MsgType("1"); }

    public TestRequest(
      QuickFix.TestReqID aTestReqID )
    : base(MsgType()) {
      set(aTestReqID);
    }

    public void set(QuickFix.TestReqID value)
    { setField(value); }
    public QuickFix.TestReqID get(QuickFix.TestReqID  value)
    { getField(value); return value; }
    public QuickFix.TestReqID getTestReqID()
    { QuickFix.TestReqID value = new QuickFix.TestReqID();
      getField(value); return value; }
    public bool isSet(QuickFix.TestReqID field)
    { return isSetField(field); }
    public bool isSetTestReqID()
    { return isSetField(112); }

  };

}

